integration_tests/tests/consistency/user_aggregated_aggs.sql

```diff
@@ -0,0 +1,63 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+with prod as (
+    select
+        sys_user_id,
+        source_relation,
+        count_distinct_sys_user_group_ids,
+        count_distinct_sys_user_role_ids,
+        count_distinct_sys_user_role_names,
+        count_distinct_included_roles,
+        count_distinct_sys_user_group_roles
+    from {{ target.schema }}_servicenow_prod.servicenow__user_aggregated
+    group by 1,2
```

Comment:
Even after prod is in sync with dev this test will still fail. We are grouping by `sys_user_id` and `source_relation`, but then not doing any aggregates. Therefore we result in an error like the following once all fields are available in prod and dev.

![image](https://github.com/user-attachments/assets/6077f256-32fc-4425-8a4c-23655ce67722)

We need to adjust this test to ensure it will work once the changes are live. I imagine doing a sum on these aggregate fields should do the trick.

Reply:
Ah we don't need the `group by` here as it's already aggregated on the `sys_user_id` level from the model. Removed that